### DISCUSSION TITLE
Welcome to PSGSuite Discussions!

### DISCUSSION THREAD

**Discussion by scrthq** [SCRT-HQ/PSGSuite#389] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phatmandrake** [comment]
Oh my god it's happening I'm excited to help out 🥳

**Comment by aw-hsid** [comment]
Hey, Systems Engineer here. I use this module a ton in automations for the company I work for. Great work btw, love being able to use this as it allows for a ton of easy automation. I do have a question about a function I'm looking to perform in one of my automations, but don't readily see a function in the module that does this.

What I am looking to do is to change the compliance settings in the GMAIL app for a specific organizational unit. More specifically, there is a "Restrict Delivery" field where you can set some settings to restrict inbound and outbound mail delivery. What I would like to be able to do is a few things:

1. Search for Address lists (defined under apps/google workspace/settings for gmail/Manage adddress lists) that contain a specified domain to see if an appropriate list already exists.
2. If not, the ability to create an address list
3. Once a valid address list is there, get the Compliance policies for a specific OU (apps/google workspace/settings for gmail/compliance).
4. Check if the policy I want is already set (in this case Restrict Delivery policy)
5. If not, the ability to modify the policy to disable inherited policies, add new policy, etc.

Does this functionality already exist in the module?

**Comment by EWC-abahmer** [comment]
Hello! As a side-bar to my normal duties, I serve as an integration specialist for the services our company uses. We have been using PSGSuite for at least 4 years (a cyber-attack in 2021 erased some of my brain memory, too!) and I am thankful for the useful automations we have made. I'm an amateur programmer so I won't dare to help update code, but I am full of suggestions (some even useful :)
